nbdime/merging/autoresolve.py

```diff
@@ -474,7 +494,8 @@ def autoresolve_decision(base, dec, strategies):
     elif isinstance(sub, list):
         decs = autoresolve_decision_on_list(dec, base, sub, strategies)
     elif isinstance(sub, string_types):
-        raise NotImplementedError()
+        sub = sub.splitlines(True)
+        decs = autoresolve_decision_on_list(dec, base, sub, strategies)
```

Comment:
Should this process the decisions back to a "string context" after resolution in "list context"? Or have you resolved that some other way?